### PR TITLE
[Multiple File Upload] Contain long file names to the visible portion of the screen

### LINKED ISSUE #8718: [Multiple File Upload] Long file names on mobile cause a small horizontal scroll

### PR DESCRIPTION
**Pull request by devin-civi**
### Description

Fixes issue where a long file name would push the "Remove file" anchor off the screen, particularly noticeable on mobile displays.

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [ ] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)
- [x] Removed the release notes section if the title is sufficient for the release notes description, or put more details in that section.
- [x] Created unit and/or browser tests which fail without the change (if possible)
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)
- [ ] Extended the README / documentation, if necessary. For user-facing features, consider updating [the user docs](https://github.com/civiform/docs). For "under-the-hood" changes or things more relevant to developers, consider updating [the dev wiki](https://github.com/civiform/civiform/wiki).

#### User visible changes

- [x] Tested on mobile view. See [mobile device mode](https://developer.chrome.com/docs/devtools/device-mode/)
- [x] Manually tested at 200% size
- [x] Manually tested with a screen reader if the feature is applicant-facing. See [screen reader testing](https://github.com/civiform/civiform/wiki/Testing#screen-reader-testing)

### Issue(s) this completes

Fixes #8718

### CODE REVIEW

**Comment by devin-civi** [comment]
* **#8821** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8821?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @devin-civi and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: browser-test/src/applicant/questions/file.test.ts [hunk 1]
```diff
@@ -445,6 +445,29 @@ test.describe('file upload applicant flow', () => {
       )
     })
 
+    test('can upload file with very long file name', async ({
```
Comment (gwendolyngoetz) [inline browser-test/src/applicant/questions/file.test.ts:448]: (blocking) We don't need a separate test just for this; it's too fine grained for the browser tests. Just add the long file name to whichever test is the primary success path of multi-file test.

**Review by gwendolyngoetz — CHANGES_REQUESTED** [review]

File: browser-test/src/applicant/questions/file.test.ts (hunk 1 above)
Comment (devin-civi) [reply browser-test/src/applicant/questions/file.test.ts]: Done.

File: browser-test/image_snapshots/file_test/file-uploaded-very-long-name.png
Comment (gwendolyngoetz) [inline browser-test/image_snapshots/file_test/file-uploaded-very-long-name.png:1]: Let's set the "remove file" `a` tag to have `text-wrap: nowrap` that way the links will be the same on each line

**Review by gwendolyngoetz — CHANGES_REQUESTED** [review]

File: browser-test/image_snapshots/file_test/file-uploaded-very-long-name.png
Comment (devin-civi) [reply browser-test/image_snapshots/file_test/file-uploaded-very-long-name.png:1]: Changed it not to shrink instead.

**Review by gwendolyngoetz — APPROVED** [review]

**Pull request merged**